### PR TITLE
feat: conditionally tokenize cells

### PR DESCRIPTION
**Pull request by chrismwendt**
Motivation: some code hosts have already tokenized the code view (e.g. Bitbucket). The tokenization currently prevents text selection from working on Bitbucket https://github.com/sourcegraph/sourcegraph/issues/4051

Implementation: this lets callers specify a boolean `tokenize` to control whether or not tokenization occurs.

Slack conversation https://sourcegraph.slack.com/archives/CHXHX7XAS/p1559599129008000

### CODE REVIEW

File: src/positions.ts [hunk 1]
```diff
@@ -26,9 +26,13 @@ export interface PositionEvent {
 }
 
 export { DOMFunctions, DiffPart }
-export const findPositionsFromEvents = (options: DOMFunctions) => (
-    elements: Subscribable<HTMLElement>
-): Observable<PositionEvent> =>
+export const findPositionsFromEvents = ({
+    domFunctions,
+    tokenize = false,
```
Comment (felixfbecker) [inline src/positions.ts]: I'd default this to `true` so it's a non-breaking change (we provide it everywhere anyway)

**Comment by felixfbecker** [comment]
Agree on needing a test.

File: src/positions.ts (hunk 1 above)
Comment (chrismwendt) [reply src/positions.ts]: ✅ 	Changed default to `true`

**Comment by chrismwendt** [comment]
✅ Added a test.

**Pull request merged**